manifests/generic.pp

```diff
@@ -18,8 +18,7 @@
       group  => 'root',
       mode   => '0644',
       # Updated 2020-03, see https://help.duo.com/s/article/5503
-      source => 'puppet:///modules/duo_unix/DUO-GPG-PUBLIC-KEY',
-      notify => Exec['Duo Security GPG Import'];
+      source => "puppet:///modules/${module_name}/DUO-GPG-PUBLIC-KEY",
```

Comment:
I'd like to recommend not doing the ${module_name} thing.
It makes sense for what we were doing internally with module shuffling in our own repos, but there are a good number of places where you already have assumed the name of the module.  So the right answer would be "either do it everywhere, or nowhere", and style-wise / people don't do it.  So I'd like to reject that line.

Reply:
SGTM. Replaced all occurrences of `${module_name}` with `duo_unix`.